Server/serverHelpers.js

```diff
@@ -63,16 +66,63 @@ const arrayOfIdsReviewGetter = (req, res, idArray) => {
           res.send(arrayOfRecommendedReviews);
         })
         .catch((error) => {
-          res.setStatus(500);
+          console.error(error);
+          res.status(500);
           res.send(error);
         })
     })
     .catch((error) => {
-      res.setStatus(500);
+      console.error(error);
+      res.status(500);
```

Comment:
I think you can actually chain these together like: res.status(500).send(error)

Reply:
That's good to know, I hadn't remembered that